runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/internal/registry/EngineWorker.java

```diff
@@ -1031,6 +1054,12 @@ public Metric resolveMetric(
         return metricGroupsByName.get(metricGroupName).supply(metricName);
     }
 
+    public LongConsumer supplyGaugeWriter(
+        long metricId)
```

Comment:
so gaguge doesn't need to have namespace and bindingId?

Reply:
It does, this is to helpfully look up the gauge engine metric which as no namespace or binding.